### ISSUE TITLE
Add a new challenge

### ISSUE THREAD

**Issue by devnet0x** [devnet0x/Starknet-Security-Challenges-Factory#2]
Add a new Cairo 1.0 challenge from ethernaut/captureTheEther or any other solidity CTF to src directory and react dapp. Remember, there must exist a isComplete function to test user success. Also add a nft image into Security-Challenges-Factory-Repo. Image must be generated with dall-e using something like: "A digital art about and astronaut dog with a <challenge_name> as pixel art and with a black space background and a few little stars".

**Comment by ccolorado** [comment]
I'll take it

**Comment by devnet0x** [comment]
Carlos, i will reassign you to issue #9

**Comment by Ayoazeez26** [comment]
Hi @devnet0x can I take this?

**Comment by devnet0x** [comment]
yes, thanks for your interest. Please tell me the name of the challenge to create a new issue for that.

**Comment by devnet0x** [comment]
ok, please comment on issue#45 so I can assign it to you.

**Comment by chiscookeke11** [comment]
I’d like to help with this.

**Comment by devnet0x** [comment]
thanks, Would you like to create a new challenge? Please provide a name for the challenge so I can create an issue for it.

**Comment by Elite-tch** [comment]
May I take care of this?

**Comment by devnet0x** [comment]
Thank you! @Elite-tch Please let me know the name of the challenge you'd like to add, and I’ll create the issue.

**Comment by TobieTom** [comment]
Could I take over this issue? I am relatively new and eager to add on to my current experience

**Comment by devnet0x** [comment]
Sure @TobieTom , please let me know the name of the challenge you'd like to add, and I’ll create the issue.

**Comment by viktrified** [comment]
Could I grab this task?

**Comment by devnet0x** [comment]
gm...sure @viktrified ,please let me know the name of the challenge you'd like to add, and I’ll create the issue.
